src/pages/RoomInvitePage.tsx

```diff
@@ -70,7 +70,9 @@ function RoomInvitePage({betas, report, policies}: RoomInvitePageProps) {
         });
         const newSelectedOptions: ReportUtils.OptionData[] = [];
         selectedOptions.forEach((option) => {
-            newSelectedOptions.push(option.login && option.login in detailsMap ? {...detailsMap[option.login], isSelected: true} : option);
+            newSelectedOptions.push(
+                option.login && option.login in detailsMap ? {...detailsMap[option.login], accountID: detailsMap[option.login].accountID ?? undefined, isSelected: true} : option,
+            );
```

Comment:
Same, is it possible to update `OptionsListUtils.MemberForList` `accountID` typing instead? It looks like removing `| null` there fix the TS problem.
https://github.com/Expensify/App/blob/a462e94d692e153e5a756c6a3306dfdd7f00e50e/src/libs/OptionsListUtils.ts#L169